src/app-layout/visual-refresh-toolbar/toolbar/styles.scss

```diff
@@ -0,0 +1,95 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../../../internal/styles' as styles;
+@use '../../../internal/styles/tokens' as awsui;
+@use '../../../internal/generated/custom-css-properties/index.scss' as custom-props;
+
+.universal-toolbar {
+  background-color: awsui.$color-background-layout-panel-content;
+  border-block-end: 1px solid awsui.$color-border-divider-panel-bottom;
+  box-sizing: border-box;
+  padding-block: 0;
+  padding-inline: awsui.$space-m;
+  position: sticky;
+  z-index: 840;
+  @include styles.with-motion {
+    transition: ease awsui.$motion-duration-refresh-only-slow;
+    transition-property: inset-block-start, opacity;
+  }
+
+  &.disable-body-scroll {
+    inset-block-start: 0px;
+  }
+
+  &.toolbar-hidden {
+    opacity: 0;
+  }
+
+  > .toolbar-container {
+    block-size: 48px;
+    align-items: center;
+    display: grid;
+    column-gap: awsui.$space-static-xs;
+    inline-size: 100%;
+    grid-template-columns: min-content auto minmax(140px, 1fr);
+    grid-template-rows: 1fr;
+
+    > .universal-toolbar-nav {
+      grid-column: 1;
+      padding-inline-end: awsui.$space-static-xxs;
+    }
+
+    > .universal-toolbar-breadcrumbs {
+      grid-column: 2;
+      background-color: transparent;
+      flex: 1 0;
+    }
+
+    > .universal-toolbar-drawers {
+      grid-column: 3;
+      column-gap: awsui.$space-static-xs;
+      display: flex;
+      justify-content: flex-end;
+    }
+  }
+}
+
+.drawers-desktop-triggers-container {
+  @include styles.styles-reset;
+  background-color: transparent;
+  box-sizing: border-box;
+  overflow-y: hidden;
+  overflow-x: hidden;
```

Comment:
Clipping the overflow impacts the focus ring appearance of the buttons. Is there a way we can get rid of that? Can be reproduced by focusing the info button [on this page](https://d21d5uik3ws71m.cloudfront.net/components/97753245901d23869c4530abadd065aa4b936694/dev-pages/index.html#/light/app-layout/default?appLayoutWidget=true).
<img width="173" alt="Screenshot 2024-06-21 at 10 46 57" src="https://github.com/cloudscape-design/components/assets/569011/94fe15e6-cea5-40ac-b8b0-8efe27d5fc7f">

Reply:
Added this to our bug bash doc. There are a lot more known bugs in this code and they will be fixed as follow up PRs